src/client_side_reply.cc

```diff
@@ -515,13 +496,7 @@ clientReplyContext::handleIMSReply(StoreIOBuffer result)
 SQUIDCEXTERN CSR clientGetMoreData;
 SQUIDCEXTERN CSD clientReplyDetach;
 
-/**
- * clientReplyContext::cacheHit Should only be called until the HTTP reply headers
- * have been parsed.  Normally this should be a single call, but
- * it might take more than one.  As soon as we have the headers,
- * we hand off to clientSendMoreData, processExpired, or
- * processMiss.
- */
+/// Only used once, to process HTTP reply headers received from Store.
```

Comment:
IIRC it may be called twice when handling Vary.

Reply:
Agreed. Fixed in 21ae6c0.